app-emacs/flycheck/flycheck-32_p20220328.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+NEED_EMACS="24.3"
+
+inherit elisp
+
+DESCRIPTION="Modern on-the-fly syntax checking extension for GNU Emacs"
+HOMEPAGE="https://www.flycheck.org/"
+COMMIT="3b5b4248074f016922c2674789d4a242528cf4c7"
+SRC_URI="https://github.com/flycheck/flycheck/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test" # test requires cask and ert-runner which are not packaged yet
```

Comment:
I'm currently packaging buttercup for tests in another package, hopefully we'll be able to get  the flycheck tests running too :)